generator/Dockerfile

```diff
@@ -1,20 +1,22 @@
-FROM golang:1.9.2
+FROM golang:latest
 
-# snmp-mibs-downloader is in the multiverse
-RUN echo "deb http://http.us.debian.org/debian stretch non-free" >> /etc/apt/sources.list
-
-RUN apt-get update && apt-get install -y \
-  build-essential \
-  libsnmp-dev \
-  snmp-mibs-downloader
+RUN apt-get update && apt-get install -y libsnmp-dev
 
+# Pull published snmp_exporter dependency so that we have access to
+# shared code in the snmp_exporter project
 RUN go get github.com/prometheus/snmp_exporter/generator
 
+WORKDIR /go/src/github.com/prometheus/snmp_exporter/generator/
+
+# Copy local generator directory so we can develop locally and test
```

Comment:
I can see this causing problems for normal use, if someone wants to do development they can build the generator by hand.